### PR TITLE
API backlog WG - API backlog call MoM -  2nd Nov 2023

### PR DESCRIPTION
**Pull request by jordonezlucena**
API backlog WG - API backlog call MoM -  2nd Nov 2023

### CODE REVIEW

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
...fied that Vodafone general position for non-OGW related APIs is not to support them. He suggests recommending API owners to follow the default path "OGW Product WS -> CAMARA", in order to get more traction and support from the rest of operators. Finally, he notes that further details on product and technical scopes are needed. <br> - Mark (GSMA): he agreed with most of Eric's comments. Clarified that network slicing APIs are discussed in the context of OPG/OPAG from network enablers perspective. Promotion within Open Gateway Product WS may come in the future, once technical capabilities are available and there is a market demand. <br> - Jose (Telefonica): he agreed with Eric's points, though it also noted that nothing prevents a company from contributing with a non-OGW API. <br> - Eric (Vodafone): he concurred, but also noted that China Unicom is a OGW MoU signatory, and one of the MoU committments is to steer work through OGW programme instead of direct submission to CAMARA. <br> - Jose (Telefonica): suggested keeping discussion off-line, and encouraged all participants to post these comments in the issue/PR thread, so API owners can receive feedback and take action accordingly. |
```
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:61]: The proposal is now in #333, as mentioned in https://github.com/camaraproject/WorkingGroups/issues/317.

Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md]: Good catch. Fixed

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+| [#294](https://github.com/camaraproject/WorkingGroups/issues/294) | Verizon| **New API proposal: Network Insights** <br> Updates: <br> - TSC asked specific and understandable API name for Network Insights. Offline discussion with GSMA and Verizon for the new API name for Network Insights. <br> - Verizon: What we have not decided yet is the name for the sub project which will have the specific contribution(yaml) tagged to. i was considering that once the project is kicked off and we have the project call we can finalize the name among the participants interested in this project. If this is holding up the whole process then lets please go ahead with "Connectivity Insights". <br> - API backlog WG will inform TSC that "Connectivity Insights" is proposed as new Sub-Project name (AP 20231019-04 cannot be closed, until TSC ok). <br> - for Sub-Project initial maintainers (AP 20231019-04 can be closed).
```
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:60]: Not explicitly discussed within TSC, but as there was no objection, I asked @MarkusKuemmerle to create the new repository as requested within this [message](https://lists.camaraproject.org/g/tsc/message/62).

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+| **Current issues** |  |  |
+| [#271](https://github.com/camaraproject/WorkingGroups/issues/271)  | China Telecom, Huawei| **New API proposal: Site to cloud (S2C) VPN**</br> Updates: <br> - Proposed for drop #3, but unfortunately didn't go shortlisted. </br> - Progress offline among interested parties (API owners, AT&T) with API backlog and TSC mediating. Good discussion and clarification on technical and product scope. <br> - TSC reported offline that they are in position of accepting the proposal, though encourage AT&T and China Telecom aligment within the Sub-Project. <br> - Eric (Vodafone): he reported that TSC gives green light to this Sub-Project, with this decision formally made on 2nd Nov call. <br> - Jose (Telefonica): he asks Pierre (AT&T) whether AT&T would join the Sub-Project, taking guidance from TSC. <br> - Pierre (AT&T): need to align internally with the AT&T colleagues who led the Catalyst, and would come back with a response (AP 20231102-02). <br> - Once the sub-project is approved, the supporters need to provide initial maintainers (AP 20231102-02).|
```
Comment (hdamker) [inline APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:59]: You might want to add a note that the TSC on Nov 2nd approved the creation of the sub project. The minutes are still in work, but please go forward to provide the maintainers.

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md (hunk 2 above)
Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md:60]: I will wait for the official confirmation from TSC to close this action point, for full transparency

File: APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md (hunk 3 above)
Comment (jordonezlucena) [reply APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md]: Done. Thank you!

**Review by hdamker — DISMISSED** [review]
My comments are addressed, thanks.

**Review by eric-murray — APPROVED** [review]

**Comment by eric-murray** [comment]
@jordonezlucena
I clarified my comments on KYC Age Verification, which is that Vodafone support including this in the existing KnowYourCustomer sub-project. We will not formally support this specific API proposal as such, so I removed that action.

**Review by MarkCornall — APPROVED** [review]
no additional comments

**Pull request merged**